hibp.js

```diff
@@ -29,15 +29,11 @@ const HIBP = {
       // Parse response body, format:
       // {breachedAccount1sha1}:{breach1Name}[,{breach2Name},...]
       // {breachedAccount2sha1}:{breach3Name}[,{breach4Name},...]
-      for (const breachedAccount of response.body.split("\n")) {
-        if (!breachedAccount) {
-          break;
-        }
-        const split = breachedAccount.split(":");
-        const breachedSha1 = split[0];
-        const breaches = split[1].split(",");
+      for (const breachedAccount of response.body.trim().split("\n")) {
+        const [breachedSha1, breachNamesStr] = breachedAccount.split(":", 2);
```

Comment:
This still doesn't work, for example, if breachedAccount is "a:b:c:d:e", split(":", 2) will return ["a", "b"] and omit the rest. What we want is ["a", "b:c:d:e"]